src/components/common/Stepper.vue

```diff
@@ -96,14 +96,6 @@ export default class Stepper extends Vue {
     return false
   }
 
-  /** show error styling for reveiw-confirm step after file and pay is selected */
-  protected isReviewStepValid (route: RouteNames): boolean {
-    return (
-      route !== RouteNames.DISSOLUTION_REVIEW_CONFIRM ||
-      this.getValidateSteps
-    )
-  }
-
```

Comment:
Please document and run a test case for this (in the ticket) to verify that you didn't break existing functionality.

Reply:
Yeah about that. I tried to add unit tests to test for different scenarios, but the icons with class `step__btn2` always shows `display: true`, meaning that there was no way I could test whether the red cross icon or the green check icon is being displayed. A little help is needed here.

Reply:
A manual test case is all I'm asking for.
1. find out what the previous code was for
2. verify that it still works

Add the test case and result to your ticket for reviewers and QA to see. Thx!

Reply:
The previous code was trying to do the same job: show error styling for reveiw-confirm step after file and pay is selected. However, it didn't and it's confusing for me. Alex will have to explain it. I left a comment in that [pull request](https://github.com/bcgov/business-create-ui/pull/418) after it was merged. There was no unit test when he added the code in his PR. For this pull request, I wanted to add unit test case, but there is some coding difficulty as mentioned above. If you are talking about test case that I run on local environment using filings, then I can confirm that the scenarios commented at the bottom of the ticket were tested in my local environment.

Reply:
I am talking about test steps that you perform manually (locally), and document in the ticket comments so that we can see you did it.

Reply:
Thanks for the clarification. I've tested various scenarios locally and added the validation scenarios in the ticket. I updated the comment just now for QA testing.